core/src/mast/mod.rs

```diff
@@ -393,6 +459,41 @@ impl MastForest {
     pub fn nodes(&self) -> &[MastNode] {
         &self.nodes
     }
+
+    // TODO: Replace with Multi iterator?
+    // /// Returns an iterator which traverses over the nodes in a depth-first search and returns
+    // nodes /// in postorder.
```

Comment:
Yeah, I'll remove it since the iterator is now no longer as general-purpose as it was before and we would have to make the item of the iterator public, which seems undesirable.